autosklearn/estimators.py

```diff
@@ -42,7 +43,8 @@ def __init__(
         logging_config=None,
         metadata_directory=None,
         metric=None,
-        scoring_functions=None,
+        scoring_functions: List[Scorer] = None,
```

Comment:
Sorry for not being clear enough earlier, could you please make this `Optional[List[Scorer]]` to explicitly state that this can be `None`?

Reply:
Done